runtime/src/main/java/org/corfudb/runtime/collections/PersistedStreamingMap.java

```diff
@@ -158,8 +154,8 @@ public boolean containsValue(@NonNull Object value) {
      */
     @Override
     public V get(@NonNull Object key) {
-        getCounter.ifPresent(Counter::increment);
-        Optional<Timer.Sample> recordSample = MicroMeterUtils.startTimer(getCounter, SAMPLING_RATE);
+        Optional<Timer.Sample> recordSample = MicroMeterUtils.startTimer(
+                SAMPLING_RATE < ThreadLocalRandom.current().nextInt(BOUND));
```

Comment:
The inequality looks like its reversed.

This will sample roughly at 69% (99 - 31), but it seems like you want it to be 30%